htlcswitch/hop/payload.go

```diff
@@ -106,7 +106,6 @@ func NewLegacyPayload(f *sphinx.HopData) *Payload {
 
 	return &Payload{
 		FwdInfo: ForwardingInfo{
-			Network:         BitcoinNetwork,
```

Comment:
Just realized this is a hard-coded value😂, and the `Realm` field for the legacy hop is never used...